docs/source/tensorbay_cli/completion.rst

```diff
@@ -0,0 +1,180 @@
+#################
+Shell Completion
+#################
+
+CLI provides tab completion support for Bash (version 4.4 and up), Zsh, and Fish. It is possible to add support for
+other shells too.
+
+Shell completion suggests command names and option names. Options are only listed if at least a dash has been entered.
+Hidden commands and options are not shown.
+
+.. code:: console
+
+    $ gas <TAB><TAB>
+    auth     -- Authenticate the accessKey of gas.
+    branch   -- List, create or delete branches.
+    commit   -- Commit drafts.
+    config   -- Configure the options when using gas CLI.
+    cp       -- Copy local data to a remote path.
+    dataset  -- List, create or delete datasets.
+    draft    -- List or create drafts.
+    log      -- Show commit logs.
+    ls       -- List data under the path.
+    rm       -- Remove the remote data.
+    tag      -- List, create or delete tags.
+    $ gas auth -<TAB><TAB>
+    --get    -g  -- Get the accesskey of the profile
+    --unset  -u  -- Unset the accesskey of the profile
+    --all    -a  -- All the auth info
+    --help       -- Show this message and exit.
+
+Activation
+***********
+
+Completion is only available if a script is installed and invoked through an entry point, not through the ``python``
```

Comment:
Can we make the description more specific? This completion is for `gas` not for general completion.